examples/time_frequency/plot_time_frequency_erds.py

```diff
@@ -96,23 +97,45 @@ def center_cmap(cmap, vmin, vmax):
 freqs = np.arange(2, 36, 1)  # frequencies from 2-35Hz
 n_cycles = freqs  # use constant t/f resolution
 vmin, vmax = -1, 1.5  # set min and max ERDS values in plot
+baseline = [-1, 0]  # baseline interval (in s)
 cmap = center_cmap(plt.cm.RdBu, vmin, vmax)  # zero maps to white
 
 for event in event_ids:
-    power = tfr_multitaper(epochs[event], freqs=freqs, n_cycles=n_cycles,
-                           use_fft=True, return_itc=False, decim=2)
-    power.crop(tmin, tmax)
+    tfr = tfr_multitaper(epochs[event], freqs=freqs, n_cycles=n_cycles,
+                         use_fft=True, return_itc=False, average=False,
+                         decim=2)
+    tfr.crop(tmin, tmax)
+    tfr.apply_baseline(baseline, mode="percent")
 
     fig, ax = plt.subplots(1, 4, figsize=(12, 4),
                            gridspec_kw={"width_ratios": [10, 10, 10, 1]})
-    for i in range(3):
-        power.plot([i], baseline=[-1, 0], mode="percent", vmin=vmin, vmax=vmax,
-                   cmap=(cmap, False), axes=ax[i], colorbar=False, show=False)
+    average_tfr = tfr.average()
+    for i in range(3):  # for each channel
+        shape = tfr.data.shape[-2:]  # (frequency, time)
+        mask = np.full(shape, False)  # initially mask all values
+
+        kwargs = dict(n_permutations=100, step_down_p=0.05, seed=1)
+        # positive clusters
+        _, c1, p1, _ = pcluster_test(tfr.data[:, i, ...], tail=1, **kwargs)
```

Comment:
use a meaningful var name (e.g., ch_idx)

Reply:
Changed to `ch` for PEP8 79 char limit reasons.

Reply:
I'd think good python style puts var names over one liners :) But `ch` should be enough.